language/src/modules/Language.rb

```diff
@@ -802,7 +807,7 @@ def MakeProposal(force_reset, language_changed)
       Builtins.foreach(Pkg.GetAdditionalLocales) do |additional|
         # add the language for both kind of values ("cs" vs. "pt_PT")
         if !Builtins.contains(langs, additional)
-          additional = "en_US" if additional == "en"
+          additional = DEFAULT_FALLBACK_LANGUAGE if additional == "en"
```

Comment:
We could consider using a hash for such mapping. But I guess is outside of the scope for this PBI.